src/components/view-item/index.js

```diff
@@ -75,9 +75,25 @@ const ViewItem = ({
         case 'weapon':
             itemIsEquipped =
                 JSON.stringify(equipped.weapon) === JSON.stringify(data);
+            // Clone the dice, because otherwise the forEach will modify the dice in the object
+            let dice = data.dice.slice();
+            let die = new Map();
+            dice.forEach(dice => {
+                die.set(dice.type, (die.get(dice.type) || 0) + 1);
+            });
+
+            let value = '';
+            die.forEach((amount, type) => {
+                if (value.length > 0) {
+                    value += ' + ';
+                }
+                value += amount + '' + type;
+            });
+            value += ' + ' + data.modifier;
```

Comment:
I'd like to see this expanded to support the full dice notation, which includes multiplication and subtraction.

Reply:
So, you're essentially wanting a parser that understands dice notation?

Reply:
I made a parser anyways. It doesn't currently support selective dice, percentile dice, or '3*(2d6 + 4)' (where, in the wikipedia page it says repeat the stuff in the brackets 3 times).

Some, if not all, technically, can be added if necessary.

Reply:
Though, if we really want to go that route, I'd recommend we look into this library: https://github.com/GreenImp/rpg-dice-roller

Reply:
Never mind, just added selective dice. I don't really see the point of the last example in the multipliers section on the wikipedia section.

There's no error checking, but I figured because it's controlled by us (at least for now), it's not a requirement.